kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue

```diff
@@ -1,16 +1,257 @@
 <template>
 
-  <audio autoplay=true src='https://freemusicarchive.org/music/download/8a46d5ff69d7c426d2a0a6854d96a683dec17802' controls=true>
-  </audio>
+  <div id="audio-wrapper">
+    <div id="play-and-time">
+      <button 
+        @click="togglePlay" 
+        class="play-button" 
+        :class="{ 'is-play': isPlay, 'is-pause': isPause }"
+        ></button>
+      <div id="current-time">
+        {{ currentMinutes }} : {{ formattedCurrentSec }}
+      </div>
+      <input 
+        v-el:timebar 
+        @click="setTimebar" 
+        class="timeline" type="range" min="0" max="100" value="0">
```

Comment:
Could you not use the `v-model` attribute here? https://vuejs.org/guide/forms.html

I think this might simplify some of your code somewhat.